internal/codeintel/autoindexing/service.go

```
@@ -428,7 +443,15 @@ func (s *Service) inferIndexJobsFromRepositoryStructure(ctx context.Context, rep
 		return nil, err
 	}
 
-	indexes, err := s.inferenceService.InferIndexJobs(ctx, api.RepoName(repoName), commit, overrideScript)
+	script, err := s.store.GetInferenceScript(ctx)
+	if err != nil {
+		s.logger.Error("failed to fetch inference script from database", log.Error(err))
+	}
```

should we bail or continue on with the remaining options? @efritz

---

This is a database-level error, right? Since script will be an empty string if not set? If so then I think we can return the error at this point.